services/app-web/src/pages/LinkYourRates/LinkRateSelect.tsx

```diff
@@ -106,19 +112,51 @@ export const LinkRateSelect = ({
         }
     }
 
+    const onInputChange = (
+        newValue: LinkRateOptionType,
+        { action }: ActionMeta<LinkRateOptionType>
+    ) => {
+        if (action === 'select-option') {
+            const linkedRateID = newValue.value
+            // const linkedRateName = newValue.label
+            const linkedRate = rates.find((rate) => rate.id === linkedRateID)
+            const linkedRateForm: FormikRateForm = convertGQLRateToRateForm(
+                getKey,
+                linkedRate
+            )
+            // put already selected fields back in place
+            linkedRateForm.ratePreviouslySubmitted = 'YES'
+            linkedRateForm.linkedRates = [
+                {
+                    rateId: linkedRateID,
+                    rateName: 'LOOK_FOR_ME', // linkedRateName,
```

Comment:
For now, are we intentionally keeping this for rateName?

Reply:
I'm a little confused on how this is used. linkedRateForm.linkedRates is not clicking for me. why does a rate have an array of linkedRates?